src/components/common/request-details.vue

```diff
@@ -45,6 +45,7 @@
               <li>{{`${contactPerson}`}}</li>
               <li>{{applicant.emailAddress}}</li>
               <li>{{applicant.phoneNumber}}</li>
+              <li><span class="label" v-if="!!getFolioNumber">Folio: </span>{{getFolioNumber}}</li>
```

Comment:
Just curious: why do you need the !! ?

Reply:
I don't think we need. It was just to explicit. I'm gonna remove it.

Reply:
OK. I know we sometimes use that to cast something else to a boolean (to satisfy property/method typing), but it's probably not needed here.